lib/src/main/java/com/hootsuite/nachos/NachoTextView.java

```diff
@@ -49,78 +49,78 @@
  * </p>
  * <h1>Validation</h1>
  * <p>
- *     This class can perform validation when certain events occur (such as losing focus). When the validation occurs is decided by
- *     {@link AutoCompleteTextView}. To perform validation, set a {@link NachoValidator}:
- *     <pre>
+ * This class can perform validation when certain events occur (such as losing focus). When the validation occurs is decided by
+ * {@link AutoCompleteTextView}. To perform validation, set a {@link NachoValidator}:
+ * <pre>
  *         nachoTextView.setNachoValidator(new ChipifyingNachoValidator());
  *     </pre>
- *     Note: The NachoValidator will be ignored if a ChipTokenizer is not set. To perform validation without a ChipTokenizer you can use
- *     {@link AutoCompleteTextView}'s built-in {@link AutoCompleteTextView.Validator Validator} through {@link #setValidator(Validator)}
+ * Note: The NachoValidator will be ignored if a ChipTokenizer is not set. To perform validation without a ChipTokenizer you can use
+ * {@link AutoCompleteTextView}'s built-in {@link AutoCompleteTextView.Validator Validator} through {@link #setValidator(Validator)}
  * </p>
  * <h1>Editing Chips</h1>
  * <p>
- *     This class also supports editing chips on touch. To enable this behavior call {@link #enableEditChipOnTouch(boolean, boolean)}. To disable this
- *     behavior you can call {@link #disableEditChipOnTouch()}
+ * This class also supports editing chips on touch. To enable this behavior call {@link #enableEditChipOnTouch(boolean, boolean)}. To disable this
+ * behavior you can call {@link #disableEditChipOnTouch()}
  * </p>
  * <h1>Example Setup:</h1>
  * <p>
- *     A standard setup for this class could look something like the following:
- *     <pre>
+ * A standard setup for this class could look something like the following:
+ * <pre>
  *         String[] suggestions = new String[]{"suggestion 1", "suggestion 2"};
-           ArrayAdapter&lt;String&gt; adapter = new ArrayAdapter&lt;&gt;(this, android.R.layout.simple_dropdown_item_1line, suggestions);
-           nachoTextView.setAdapter(adapter);
-           nachoTextView.addChipTerminator('\n', ChipTerminatorHandler.BEHAVIOR_CHIPIFY_ALL);
-           nachoTextView.addChipTerminator(' ', ChipTerminatorHandler.BEHAVIOR_CHIPIFY_TO_TERMINATOR);
-           nachoTextView.setIllegalCharacters('@');
-           nachoTextView.setNachoValidator(new ChipifyingNachoValidator());
-           nachoTextView.enableEditChipOnTouch(true, true);
-           nachoTextView.setOnChipClickListener(new NachoTextView.OnChipClickListener() {
-              {@literal @Override}
-               public void onChipClick(Chip chip, MotionEvent motionEvent) {
-                   // Handle click event
-               }
-           });
+ * ArrayAdapter&lt;String&gt; adapter = new ArrayAdapter&lt;&gt;(this, android.R.layout.simple_dropdown_item_1line, suggestions);
```

Comment:
⛔️  can this part still have `<pre>` and not kill all the nice indenting needed?